app/src/androidTestE2eTest/java/io/appium/uiautomator2/unittest/test/internal/TestUtils.java

```diff
@@ -165,6 +165,8 @@ public static JSONObject convertByToJson(By by, String contextId) {
                 strategy = "id";
             } else if (by instanceof By.ByXPath) {
                 strategy = "xpath";
+            } else if (by instanceof By.ByText) {
```

Comment:
do we need one more selector with the same target?

Reply:
**By.ByText** looks into the **text** property of a UI element. None of the other selectors seem to do that. Accept as is.

Reply:
then it makes sense to simply add it to the next entry using `||`